[quant][graphmode] Remove quantizeBias

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #26390 [quant][graph] PyTorch Graph Mode Quantization API
* #26389 [quant][graph] Add inplace argument to InsertObservers and InsertQuantDeQuant
* #26425 [quant][graphmode] Get scalar type from observer module
* #26415 [quant][graphmode] Remove quantization for bias in pattern
* #26414 [quant][graphmode] Fix quantized::linear QuantFusion patterns
* **#26388 [quant][graphmode] Remove quantizeBias**

Summary:
We don't need to quantize bias after https://github.com/pytorch/pytorch/pull/26057

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17465554](https://our.internmc.facebook.com/intern/diff/D17465554)

jerryzh168 committed:
[quant][graphmode] Remove quantizeBias

Summary:
We don't need to quantize bias after https://github.com/pytorch/pytorch/pull/26057

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Remove quantizeBias"


Summary:
We don't need to quantize bias after https://github.com/pytorch/pytorch/pull/26057

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM commented on torch/csrc/jit/passes/quantization.cpp:
```diff
@@ -601,12 +568,7 @@ void InsertQuantDeQuantImpl(
-        } else {
-          qh.quantizeTensor(v);
-        }
+        qh.quantizeTensor(v);
```
It seems that after this change we will be quantizing bias - we would just do that in the same manner as a usual tensor. Is it intended or do we want to skip bias tensors completely?

jerryzh168 replied:
oh, I'll restore the check

jerryzh168 replied:
to support this properly we'll probably need a value set as well.. but we can do this later

jerryzh168 committed:
Update on "[quant][graphmode] Remove quantizeBias"


Summary:
We don't need to quantize bias after https://github.com/pytorch/pytorch/pull/26057

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Remove quantizeBias"


Summary:
We don't need to quantize bias after https://github.com/pytorch/pytorch/pull/26057

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Remove quantizeBias"


Summary:
We don't need to quantize bias after https://github.com/pytorch/pytorch/pull/26057

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
👍

jerryzh168 committed:
Update on "[quant][graphmode] Remove quantizeBias"


Summary:
We don't need to quantize bias after https://github.com/pytorch/pytorch/pull/26057

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Remove quantizeBias"


Summary:
We don't need to quantize bias after https://github.com/pytorch/pytorch/pull/26057

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17465554](https://our.internmc.facebook.com/intern/diff/D17465554)

[ghstack-poisoned]
